### PR TITLE
fix: issue where the step and workflow editor crash when building

### PR DESCRIPTION
**Pull request by Gusinuhe**
### Description

**Fixes**: The Step Inspector and the Workflow Editor no longer crash when Unity is building.

### Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

### How Has This Been Tested?

Build something with those windows open.

### CODE REVIEW

File: Editor/UI/Windows/CourseWindow.cs [hunk 1]
```diff
@@ -231,15 +231,18 @@ private void DrawChapterWorkflow(Rect scrollRect)
 
         private void OnSceneOpened(Scene scene, OpenSceneMode mode)
         {
-            if (RuntimeConfigurator.Exists == false)
+            if (BuildPipeline.isBuildingPlayer == false && RuntimeConfigurator.Exists == false)
```
Comment (burnedikt) [inline Editor/UI/Windows/CourseWindow.cs:234]: Shouldn't this be an "or" case?

Comment (Gusinuhe) [reply Editor/UI/Windows/CourseWindow.cs:234]: No, the idea is to close the window only when a new empty scene is opened and Unity is not building.

For some reason, Unity opens a new scene when running unit test and when building, after it finishes those proceses it goes back to the original scene 🤷‍♂️🤦‍♂️

**Review by burnedikt — APPROVED** [review]

**Comment by SimonTheSourcerer** [comment]
tested, works for me

**Review by SimonTheSourcerer — APPROVED** [review]

**Pull request merged**